diff -u gxsnmp-0.0.16.orig/gxsnmp.desktop gxsnmp-0.0.16/gxsnmp.desktop
--- gxsnmp-0.0.16.orig/gxsnmp.desktop	Fri Dec 14 15:24:07 2001
+++ gxsnmp-0.0.16/gxsnmp.desktop	Thu Dec 25 01:15:59 2003
@@ -4,22 +4,24 @@
 Name[de]=SNMP Monitor
 Name[es]=Monitor SNMP
 Name[fr]=Moniteur SNMP
-Name[ja]=SNMP•‚•À•ø
+Name[ja]=SNMP„É¢„Éã„Çø
 Name[no]=SNMP-Monitor
+Name[pl]=Monitor SNMP
 Name[pt]=Monitor SNMP
 Name[wa]=Waitieu SNMP
-Name[pl]=Monitor SNMP
 Comment=SNMP Monitor
-Comment[ca]=Monitor i administraciÛ SNMP (Simple Network Management Protocol)
+Comment[ca]=Monitor i administraci√≥ SNMP (Simple Network Management Protocol)
 Comment[de]=SNMP Monitor
-Comment[es]=Monitor y administraciÛn SNMP (Simple Protocolo de GestiÛn de Redes)
-Comment[fr]=Moniteur et administration SNMP (Protocole Simple de Gestion de RÈseaux)
-Comment[ja]=SNMP¥…Õ˝•‚•À•ø(Simple Network Management Protocol)
+Comment[es]=Monitor y administraci√≥n SNMP (Simple Protocolo de Gesti√≥n de Redes)
+Comment[fr]=Moniteur et administration SNMP (Protocole Simple de Gestion de R√©seaux)
+Comment[ja]=SNMPÁÆ°ÁêÜ„É¢„Éã„Çø(Simple Network Management Protocol)
 Comment[no]=SNMP-Monitor
+Comment[pl]=Monitorowanie i administracja SNMP (Simple Network Management Protocol)
 Comment[pt]=Monitor SNMP
-Comment[wa]=Waitieu eyËt manaedjmint SNMP (Simpe Protocole di Manaedjmint di Rantoeles)
-Comment[pl]=Monitor SNMP
+Comment[wa]=Waitieu ey√®t manaedjmint SNMP (Simpe Protocole di Manaedjmint di Rantoeles)
 Exec=gxsnmp
 Icon=gnome-networktool.png
-Terminal=0
+Terminal=false
 Type=Application
+Encoding=UTF-8
+# vi: encoding=utf-8
