diff -Nru gxsnmp-0.0.15.1/docs/C/Makefile.am gxsnmp-0.0.15.1.new/docs/C/Makefile.am
--- gxsnmp-0.0.15.1/docs/C/Makefile.am	Tue Jun 29 03:15:13 1999
+++ gxsnmp-0.0.15.1.new/docs/C/Makefile.am	Sun Feb  6 23:10:10 2000
@@ -33,13 +33,13 @@
 	-cp images/*.jpg $(distdir)/images
 
 install-data-local: gxsnmp.html
-	$(mkinstalldirs) $(gxsnmp_helpdir)/images
+	$(mkinstalldirs) $(DESTDIR)$(gxsnmp_helpdir)/images
 	-for file in $(srcdir)/gxsnmp/*.html $(srcdir)/gxsnmp/*.css; do \
 	basefile=`basename $$file`; \
-	$(INSTALL_DATA) $(srcdir)/$$file $(gxsnmp_helpdir)/$$basefile; \
+	$(INSTALL_DATA) $(srcdir)/$$file $(DESTDIR)$(gxsnmp_helpdir)/$$basefile; \
 	done
 	-for file in $(srcdir)/images/*.jpg; do \
-	$(INSTALL_DATA) $(srcdir)/$$file $(gxsnmp_helpdir)/images;\
+	$(INSTALL_DATA) $(srcdir)/$$file $(DESTDIR)$(gxsnmp_helpdir)/images;\
 	done
 
 gxsnmp.ps: gxsnmp.sgml
